### PR TITLE
added Norwegian (Bokmål)

### PR DESCRIPTION
**Pull request by arnotixe**
hope I did this right - forked master, pushed to my own fork and then created PR

### CODE REVIEW

File: po/nb.po [hunk 1]
```diff
@@ -0,0 +1,606 @@
+# Finnish translation for gnome-pomodoro
+# Copyright (c) 2020 gnome-pomodoro contributors
+# This file is distributed under the same license as the gnome-pomodoro package.
+# Tuomas Jaakola <tuomas.jaakola@iki.fi>, 2020.
```
Comment (iqqmuT) [inline po/nb.po:4]: Maybe you can change the name :)

File: po/nb.po [hunk 2]
```diff
@@ -0,0 +1,606 @@
+# Finnish translation for gnome-pomodoro
```
Comment (iqqmuT) [inline po/nb.po:1]: Replace `Finnish` with `Norwegian`

**Comment by iqqmuT** [comment]
You can drop the file `po/nb.mo`, it's redundant in this repo.

**Comment by arnotixe** [comment]
:) I'll have a look

File: po/nb.po [hunk 3]
```diff
@@ -0,0 +1,606 @@
...
+"POT-Creation-Date: 2017-09-22 20:00+0200\n"
+"PO-Revision-Date: 2020-08-16 18:07-0500\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.0.6\n"
+"Last-Translator: Arno Teigseth <arnotixe@gmail.com>\n"
+"Language: no\n"
```
Comment (kamilprusko) [inline po/nb.po:19]:
```suggestion
"Language: nb\n"
```

**Comment by kamilprusko** [comment]
Looks great! Thank you @arnotixe and @iqqmuT for the review.

**Pull request merged**